Nos/Views/NoteComposer/ComposerActionBar.swift

```diff
@@ -68,7 +70,15 @@ struct ComposerActionBar: View {
         .onChange(of: expirationTime) { _, _ in
             subMenu = .none
         }
-        .alert(unwrapping: $alert) { (_: AlertAction?) in
+        .alert(unwrapping: $alert) { action in
+            switch action {
+            case .getAccount:
+                if let url = URL(string: "https://nostr.build/plans") {
```

Comment:
When I follow this link, it adds the trailing slash, so I think it'd be great if we just do it here. Not sure how much it matters, but it seems like a nice touch.
```suggestion
                if let url = URL(string: "https://nostr.build/plans/") {
```